Reorganize Get started section (#53)
doc-detective/doc-detective.github.io: docs/get-started/sample-tests.md

## Get sample files

To get a few samples, [view the samples on GitHub](https://github.com/doc-detective/doc-detective/tree/main/samples) or clone the repo and navigate to the `samples` directory:

```bash
git clone https://github.com/doc-detective/doc-detective.git
cd doc-detective/samples
npm i
```

## Run tests

To run your tests, use the `runTests` command:

```bash
npx doc-detective runTests
```

By default, Doc Detective scans the current directory for valid tests, but you can specify your test file with the `--input` argument. For example, to run tests in a file named `doc-content-inline-tests.md`, run the following command:

```bash
npx doc-detective runTests --input doc-content-inline-tests.md
```

To customize your test, file type, and directory options, create a `.doc-detective.json` [config](/docs/references/schemas/config) file. If a `.doc-detective.json` file exists in the directory when you run the comment, Doc Detective loads the config. Otherwise, you can specify a config path with the `--config` argument.

```bash
npx doc-detective runTests --config .doc-detective.json
```

> **Note**: All paths are relative to the current working directory, regardless where the config file is located.

Author: ```suggestion
**Note**: All paths are relative to the current working directory, regardless of the config file's location.
```
